charts/alfresco-repository/templates/ingress.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
   annotations:
     checkov.io/skip1: CKV_K8S_153=We're filtering out snippet in named template
     {{- include "alfresco-common.nginx.annotations" .Values }}
-    {{- include "alfresco-common.nginx.secure.annotations" .Values }}
+    {{- include "alfresco-repository.nginx.secure.annotations" . | nindent 4 }}
```

Comment:
I think it needs to be moved before the user annotations or users can't never override it, make sure to test it as well